### ISSUE TITLE
Manga Ship is broken

### ISSUE THREAD

**Issue by kyoyacchi** [keiyoushi/tachiyomi-extensions-source#1100]
### Source information

Manga Ship

### Source language

Turkish

### Steps to reproduce

1. Install Manga Ship extension
2. Try to use it.

### Expected behavior

It should load all manga correctly.

### Actual behavior

It doesn't load thumbnails, and chapters.
![Screenshot_20240207-185211](https://github.com/keiyoushi/extensions-source/assets/63583961/603c298a-4ead-4a90-9aa8-858889728f83)

The website is working fine (https://mangaship.net/)

### Mihon/Tachiyomi version

1.7.4

### Android version

Android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by SnakeDoc83** [comment]
Getting the thumbnails and chapters can be fixed but it looks like you'll have to log in to actually read the chapters (you get placeholder images if you're not logged in).

**Comment by kyoyacchi** [comment]
> Getting the thumbnails and chapters can be fixed but it looks like you'll have to log in to actually read the chapters (you get placeholder images if you're not logged in).

Yeah i just noticed that it has an image says "BÖLÜMÜ OKUMAK İÇİN GİRİŞ YAPIN" which means "Login to read the chapter".


So, will this extension get removed?

**Comment by SnakeDoc83** [comment]
> So, will this extension get removed?

Not right now, but can't say if or when someone will fix this.

**Comment by dngonz** [comment]
About this PR, I fixed the issues with the thumbnails and chapters but I didn't manage to create an account to actually fix the extension, verification email never send.
But as other older comments said there is no point on have it if no one can use it.

**Comment by Norsze** [comment]
@dngonz doesn't seem like you need to verify the email, you can just login after going through the registration, just tried it myself.
